lecture2gether-vue/src/components/VideoMetaData.vue

```diff
@@ -1,15 +1,28 @@
 <template>
-    <div class="video-meta-data" style="visibility: hidden;">
-        <div class="title">
+    <div class="video-meta-data pa-6">
+        <div class="video-title display-1"
+             :class="{ hidden: videoMetaData.title === null }"
+        >
             {{ videoMetaData.title }}
         </div>
-        <a class="creator" :href="videoMetaData.creatorLink">
-            {{ videoMetaData.creator }}
-        </a>
-        <div class="date">
-            {{ videoDate }}
+        <div>
+            <a class="creator body-1 meta-data-link"
+               :href="videoMetaData.creatorLink"
+               :class="{ hidden: videoMetaData.creator === null }"
+            >
+                {{ videoMetaData.creator }}
+            </a>
+            <div class="date body-1"
+                 :class="{ hidden: videoMetaData.date === null }"
+            >
+                {{ videoDate }}
+            </div>
         </div>
-        <a class="license" :href="videoMetaData.licenseLink">
+        <v-spacer></v-spacer>
+        <a class="license body-1 meta-data-link"
+           :href="videoMetaData.licenseLink"
+           :class="{ hidden: videoMetaData.license === null }"
```

Comment:
Instead of conditionally setting the hidden class, you should use [v-if](https://vuejs.org/v2/guide/conditional.html)

Reply:
Is resolved now :)